### PR TITLE
Hazelcast 2nd level cache + Cucumber test failing #2553

### PR DESCRIPTION
**Pull request by lordlothar99**
#2553 splitted cucumber / not cucumber tests execution

### CODE REVIEW

**Comment by deepu105** [comment]
Is it required in gradle? cc @atomfrede

**Comment by atomfrede** [comment]
I think there was already a fix for gradle. It was only failing with maven.
Am 09.01.2016 2:15 vorm. schrieb "Deepu k Sasidharan" <
notifications@github.com>:

**Comment by pascalgrimaud** [comment]
Yes, Gradle was fixed here https://github.com/jhipster/generator-jhipster/pull/2275

File: app/templates/_pom.xml [hunk 1]
```diff
@@ -694,6 +694,28 @@
...
+                <executions>
+                    <execution>
+                        <id>default-test</id>
+                        <configuration>
+                            <excludes>
+                                <exclude>**/Abstract*.java</exclude>
+                                <exclude>**/*Cucumber*.java</exclude>
+                            </excludes>
+                        </configuration>
+                    </execution>
```
Comment (pascalgrimaud) [inline app/templates/_pom.xml]: it works for all current config, but shouldn't we add this new code only for cucumber ?

Comment (jdubois) [inline app/templates/_pom.xml]: Yes, this should only be added for Cucumber

**Comment by jdubois** [comment]
Please have a look at the comment: you need to make a switch so that this is only generated for Cucumber

And then I'll merge this, of course!

**Comment by lordlothar99** [comment]
Sure!  I'll do that today

**Comment by jdubois** [comment]
Cool, let's do this :-)

**Pull request merged**